src/select.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import JWSelect from 'react-select';
+import classNames from 'classnames';
+import identity from './helpers/identity';
+
+export default class Select extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: props.defaultValue
```

Comment:
What's the use case for hard coded select with permanent value i.e. `<Select value='x' />`?